[2022-08-12 21:18:28,800] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ETL_Raízen.convert_xls_into_xlsx manual__2022-08-12T21:18:19.980399+00:00 [queued]>
[2022-08-12 21:18:28,814] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: ETL_Raízen.convert_xls_into_xlsx manual__2022-08-12T21:18:19.980399+00:00 [queued]>
[2022-08-12 21:18:28,815] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-08-12 21:18:28,817] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-08-12 21:18:28,817] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-08-12 21:18:28,837] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): convert_xls_into_xlsx> on 2022-08-12 21:18:19.980399+00:00
[2022-08-12 21:18:28,843] {standard_task_runner.py:52} INFO - Started process 3489 to run task
[2022-08-12 21:18:28,849] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_Raízen', 'convert_xls_into_xlsx', 'manual__2022-08-12T21:18:19.980399+00:00', '--job-id', '240', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Raizen.py', '--cfg-path', '/tmp/tmpaihq4qbg', '--error-file', '/tmp/tmpan36gz0w']
[2022-08-12 21:18:28,850] {standard_task_runner.py:80} INFO - Job 240: Subtask convert_xls_into_xlsx
[2022-08-12 21:18:28,925] {task_command.py:369} INFO - Running <TaskInstance: ETL_Raízen.convert_xls_into_xlsx manual__2022-08-12T21:18:19.980399+00:00 [running]> on host 4bbb34367c41
[2022-08-12 21:18:29,019] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ETL_Raízen
AIRFLOW_CTX_TASK_ID=convert_xls_into_xlsx
AIRFLOW_CTX_EXECUTION_DATE=2022-08-12T21:18:19.980399+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-12T21:18:19.980399+00:00
[2022-08-12 21:18:29,021] {ETL_Raizen.py:47} INFO - Converting file .xls into .xlsx using libreoffice
[2022-08-12 21:18:34,957] {ETL_Raizen.py:49} INFO - File .xls succesfully converted into .xlsx
[2022-08-12 21:18:34,959] {python.py:173} INFO - Done. Returned value was: None
[2022-08-12 21:18:34,986] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ETL_Raízen, task_id=convert_xls_into_xlsx, execution_date=20220812T211819, start_date=20220812T211828, end_date=20220812T211834
[2022-08-12 21:18:35,022] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-12 21:18:35,083] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
